src/Traits/Categories/ButtonHelpers.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+use NotificationChannels\OneSignal\OneSignalButton;
+use NotificationChannels\OneSignal\OneSignalWebButton;
+
+trait ButtonHelpers
+{
+    /**
+     * Add a web button to the message.
+     *
+     * @param OneSignalWebButton $button
+     *
+     * @return $this
+     */
+    public function webButton(OneSignalWebButton $button)
+    {
+        return $this->setParameter('web_buttons', [$button->toArray()]);
```

Comment:
Something is up here. Can you still chain these calls. I'm pretty sure you could before - haven't tested this - but wasn't sure if this works as expected. I'm probably completely wrong

Reply:
Actually i goes after the documentation: https://documentation.onesignal.com/reference#section-action-buttons

Doesn't tested it actually in a real world application (just run the unit tests)

Reply:
Totally. I'm just wondering if this code can chain buttons.

Previously you could do this:

```php
$this->message->webButton($button1)->webButton($button2);
```

I wasn't sure if this still worked - I might write a test to make sure you can still chain them. It's probably totally fine. I'll post the test here and let you know the result

Reply:
Hey, I've written the test and PR'd it: https://github.com/laravel-notification-channels/onesignal/pull/60

This test passes on the current release, but not this PR.